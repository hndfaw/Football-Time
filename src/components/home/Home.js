import React from 'react';
import './home.css';
import homepageImg from '../../images/welcome-page.jpg'

export const Home = () => {
  return (
    <div>
      <h2 className="welcome">Welcome to Football Time! </h2>
      <img src={homepageImg} className="homepage-img" alt="homepage img" />
    </div>
  )
}

export default Home