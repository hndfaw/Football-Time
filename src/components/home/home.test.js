import React from 'react';
import { shallow } from 'enzyme';
import { Home } from './Home';



describe('Home', () => {
  
    it('should match the match snapshot', () => {
      let wrapper = shallow(<Home  />)
      expect(wrapper).toMatchSnapshot()
    })
})