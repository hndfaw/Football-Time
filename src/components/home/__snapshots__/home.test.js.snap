// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match the match snapshot 1`] = `
<div>
  <h2
    className="welcome"
  >
    Welcome to Football Time! 
  </h2>
  <img
    alt="homepage img"
    className="homepage-img"
    src="welcome-page.jpg"
  />
</div>
`;
